src/main/java/com/atlassian/bitbucket/jenkins/internal/model/BitbucketWebhookSupportedEvents.java

```diff
@@ -0,0 +1,22 @@
+package com.atlassian.bitbucket.jenkins.internal.model;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.Set;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class BitbucketWebhookSupportedEvents {
+
+    private Set<String> applicationWebHooks;
+
+    @JsonCreator
+    public BitbucketWebhookSupportedEvents(@JsonProperty(value = "application-webhooks") Set<String> applicationWebHooks) {
+        this.applicationWebHooks = applicationWebHooks;
```

Comment:
Should we `requireNonNull` here or anything? Some kind of validation on the input